I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00keymap.rpy", line 411, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 411, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 430, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
AttributeError: 'NoneType' object has no attribute 'stores'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\bootstrap.py", line 313, in bootstrap
    renpy.main.main()
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\main.py", line 440, in main
    renpy.game.script.report_duplicate_labels()
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\script.py", line 907, in report_duplicate_labels
    if renpy.parser.report_parse_errors():
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\parser.py", line 2787, in report_parse_errors
    renpy.display.error.report_parse_errors(full_text, error_fn)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\error.py", line 187, in report_parse_errors
    error_fn=error_fn,
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\game.py", line 281, in invoke_in_new_context
    return callable(*args, **kwargs)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\error.py", line 45, in call_exception_screen
    return renpy.ui.interact(mouse="screen", type="screen", suppress_overlay=True, suppress_underlay=True)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ui.py", line 289, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\core.py", line 2672, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\core.py", line 3132, in interact_core
    renpy.exports.reload_script()
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\exports.py", line 1651, in reload_script
    renpy.game.call_in_new_context("_save_reload_game")
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\game.py", line 333, in call_in_new_context
    return renpy.execution.run_context(False)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\execution.py", line 876, in run_context
    context.run()
  File "renpy/common/00keymap.rpy", line 411, in script
    python hide:
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ast.py", line 881, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\python.py", line 1913, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/00keymap.rpy", line 411, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 430, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\loadsave.py", line 397, in save
    roots = renpy.game.log.freeze(None)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\python.py", line 1810, in freeze
    self.complete(False)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\python.py", line 1440, in complete
    self.current.stores[name], self.current.delta_ebc[name] = sd.get_changes(begin)
AttributeError: 'NoneType' object has no attribute 'stores'

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
SBU Senior Project 1.0
Thu Feb 07 01:26:18 2019
